classes/cron_processor.php

```diff
@@ -42,6 +42,9 @@ class cron_processor implements processor {
     /** @var int */
     protected $samplems;
 
+    /** @var bool */
+    protected static $isinsideoninterval = false;
```

Comment:
I find this a bit of a mouthful (eyefull??) to read, how about something simpler like $processing or $busy or $alreadyprofiling?

Reply:
fixed